### PR TITLE
chore: add utility method for object relations

### PR DESCRIPTION
**Pull request by miparnisari**

### CODE REVIEW

File: pkg/tuple/tuple.go [hunk 1]
```diff
@@ -55,6 +55,14 @@ func BuildObject(objectType, objectID string) string {
 	return fmt.Sprintf("%s:%s", objectType, objectID)
 }
 
+// GetObjectRelationAsString returns a string like "object#relation". If there is no relation it returns "object"
+func GetObjectRelationAsString(objectRelation *openfgapb.ObjectRelation) string {
```
Comment (craigpastro) [inline pkg/tuple/tuple.go:59]: You are introducing a new function, but not using it anywhere?

Comment (jon-whit) [inline pkg/tuple/tuple.go:59]: Please add some tests for this 👍

**Review by jon-whit — CHANGES_REQUESTED** [review]
Please add some tests.

File: pkg/tuple/tuple.go (hunk 1 above)
Comment (miparnisari) [reply pkg/tuple/tuple.go:59]: It's used in Auth0 FGA :)

Comment (craigpastro) [reply pkg/tuple/tuple.go:59]: Thank you! Yes, we talked offline about it. My personal opinion is that in that case it shouldn't be here, but I will defer to others.

Comment (jon-whit) [reply pkg/tuple/tuple.go:59]: There are other places in our code within OpenFGA that should delegate to this function as well. Here is a list of them that I found from a quick search (there may be others):

* `pkg/server/commands/connected_objects.go` (line 400)
* `pkg/storage/memory/memory.go` (line 387)

Please refactor those to use this method as well. Then this debate is non-applicable 😄

Comment (miparnisari) [reply pkg/tuple/tuple.go:59]: Done!  Thanks - i don't know why I didn't check where I could use this here, that was dumb..

**Review by jon-whit — APPROVED** [review]

**Pull request merged**